heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/HeronExecutorTask.java

```diff
@@ -133,6 +133,7 @@ public HeronExecutorTask(final REEFFileNames fileNames,
         env,
         topologyJar,
         topologyDefFile,
+        false,
```

Comment:
There is no verbose option in [HeronExecutorTask's field](https://github.com/objmagic/heron/blob/6543f5f531d86521f341753f821dc89dc1fe69ff/heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/HeronExecutorTask.java#L85-L94).